python/gtsam_py/python/examples/PlanarSLAMExample.py

```diff
@@ -13,25 +13,24 @@
 
 from __future__ import print_function
 
-import gtsam
 import numpy as np
-from gtsam import symbol_shorthand_L as L
-from gtsam import symbol_shorthand_X as X
+
+import gtsam
```

Comment:
This script changed significantly and has to be re-ported

Reply:
I think this one is the same?

Reply:
No. Uses shorthand now

Reply:
done